dev-docs/bidders/adnuntius.md

```diff
@@ -5,7 +5,7 @@ description: Prebid Adnuntius Bidder Adaptor
 pbjs: true
 pbs: true
 biddercode: adnuntius
-media_types: banner
+media_types: banner, video
```

Comment:
while you are at it. Could you add those meta fields as well?

- `safeframes_ok`: true/false
- `multiformat_supported`: will-bid-on-any, will-bid-on-one, will-not-bid
- `ortb_blocking_supported`: true/partial/false
- `floors_supported`: true/false

See https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter for a full reference.

Reply:
Sure!